extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -58,27 +67,53 @@ export class SubscriptionService {
   initialize_() {
     const pageConfigResolver = new PageConfigResolver(this.ampdoc_.win);
 
-    // TODO(@prateekbh): read this config from the document.
-    const platformConfigs = [
-      {
-        paywallUrl: '/subscription/1/entitlements',
-      },
-      {
-        paywallUrl: '/subscription/2/entitlements',
-      },
-    ];
-
-    return pageConfigResolver.resolveConfig().then(pageConfig => {
+    const configPromises = Promise.all([
+      this.getServiceConfig_(),
+      pageConfigResolver.resolveConfig(),
+    ]);
+
+    return configPromises.then(promiseValues => {
+      const serviceConfig = user().assert(promiseValues[0]);
```

Comment:
You have to add typing for `serviceConfig` and `pageConfig`.

Reply:
Like a /** typedef */ or a just /** {!PageConfig} *? and /** JsonObject<string, string> */

Reply:
`/** @type {!PageConfig} */`

Reply:
Done